style/battle.css

```diff
@@ -630,7 +630,7 @@ License: GPLv2
 	background: #333333;
 	color: #CCCCCC;
 }
-.dark .battle-log .chat strong,
+.dark .battle-log .chat > strong,
```

Comment:
Well that explains why it wasn't used anywhere else, but it doesn't seem to be written down anywhere.
Anyway, I used it because it needs to differentiate between ``<strong>`` that's part of the battle ("abc's team:" and updated elo score - should be blue) and ``<strong>`` that's part of a broadcast box, like for modchat (should be white). As far as I can tell, the options are:
1. use really really specific rules to override it (.battle-log .inner .chat .broadcast-red strong, once for every broadcast class)
2. use ``:not`` to exclude broadcast boxes - I couldn't get this to work (and based on its absence in the rest of the code it's probably against the style guidelines anyway)
3. use ``>`` to enforce that only ``<strong>`` tags that are direct children of ``.chat`` become blue, which is only true for the team/elo but not for broadcast boxes
Out of those, 3 seemed the least ugly to me. I suppose it'll have to be option 1?